apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -244,6 +244,30 @@ private class CredentialWrapper : ITokenAccess
         }
 
         [Fact]
+        public async Task ReadAllowsNewApis()
+        {
+            // ReSharper disable once RedundantAssignment
+            int rowsRead = -1;
```

Comment:
thats a good suggestion and would make it clearer.  a few tests use this pattern.  i'll get around to cleaning them up a little in the future.